validator/remote/src/main/java/tech/pegasys/teku/validator/remote/RemoteValidatorApiHandler.java

```diff
@@ -414,18 +414,17 @@ public void subscribeToPersistentSubnets(final Set<SubnetSubscription> subnetSub
   }
 
   @Override
-  public void prepareBeaconProposer(
+  public SafeFuture<Void> prepareBeaconProposer(
       final Collection<BeaconPreparableProposer> beaconPreparableProposers) {
-    sendRequest(
-            () ->
-                apiClient.prepareBeaconProposer(
-                    beaconPreparableProposers.stream()
-                        .map(
-                            pbp ->
-                                new tech.pegasys.teku.api.schema.bellatrix.BeaconPreparableProposer(
-                                    pbp.getValidatorIndex(), pbp.getFeeRecipient()))
-                        .collect(toList())))
-        .finish(error -> LOG.error("Failed to prepare beacon proposers", error));
+    return sendRequest(
+        () ->
+            apiClient.prepareBeaconProposer(
+                beaconPreparableProposers.stream()
+                    .map(
+                        bpp ->
```

Comment:
could probably call the variable something more like 'proposers' and it'd read more easily...